docs/standard/data/sqlite/backup.md

```diff
@@ -1,12 +1,12 @@
 ---
-title: Online Backup
+title: Online backup
 ms.date: 12/13/2019
 description: Learn how to use SQLite's online backup feature.
 ---
-# Online Backup
+# Online backup
 
-SQLite can back up database files while the app is running. This functionality is available in Microsoft.Data.Sqlite as the [BackupDatabase](/dotnet/api/microsoft.data.sqlite.sqliteconnection.backupdatabase) method on SqliteConnection.
+SQLite can back up database files while the app is running. This functionality is available in `Microsoft.Data.Sqlite` as the <xref:Microsoft.Data.Sqlite.SqliteConnection.BackupDatabase%2A> method on `SqliteConnection`.
```

Comment:
Should the product name really be an inline code block (wrapped in backticks)? I can see doing it when referring to the namespace, but I don’t think I ever do that.

Reply:
I can remove it from the product name

```suggestion
SQLite can back up database files while the app is running. This functionality is available in Microsoft.Data.Sqlite as the <xref:Microsoft.Data.Sqlite.SqliteConnection.BackupDatabase%2A> method on `SqliteConnection`.
```